includes/oauthhelper/oauth-service-actions.php

```diff
@@ -0,0 +1,209 @@
+<?php
+/**
+ * Oauth Ajax
+ *
+ * @package SimpleCalendar\Admin
+ */
+namespace SimpleCalendar\Admin;
+
+if (!defined('ABSPATH')) {
+	exit();
+}
+
+/**
+ * Oauth Ajax.
+ *
+ * @since 3.0.0
+ */
+class Oauth_Ajax
+{
+	/*
+	 *Auth site URL
+	 */
+	public static $url = SIMPLE_CALENDAR_OAUTH_HELPER_AUTH_DOMAIN . 'wp-json/api-test/v1/';
```

Comment:
Are we using `api-test` as path? Feels wrong.

Reply:
chane it to api-oauth-helper